lib/cosmos/gui/line_graph/line_graph_popups.rb

```diff
@@ -43,10 +43,11 @@ def add_popups_for_lines(x_value, value_delta)
         graph_x = scale_value_to_graph_x(x)
         graph_y = scale_value_to_graph_y(y, axis)
 
-        x_text = convert_x_value_to_text(x_text.to_f, @max_x_characters)
         if @left_button_pressed
+          x_text = convert_x_value_to_text(x, 25, true) unless x_text
```

Comment:
I defaulted the size to a big 25 characters when the mouse is pressed for popups which seems to cover all cases well.